src/select.coffee

```diff
@@ -107,6 +107,21 @@ Ember.AddeparMixins.ResizeHandlerMixin,
   # possibly says 'create item' or something along that line
   selectMenuView: null
 
+  # a map of accepted keys to show dropdown when being pressed
+  acceptedKeys: null
+
+  # set up a map for accepted keys from multiple ranges of them
+  # these are keys to show dropdown when being pressed
+  createAcceptedKeySet: ->
```

Comment:
why not make this a computed property? `acceptedKeys: Ember.computed -> ... .property()`